icu4c/source/test/intltest/itrbnf.cpp

```diff
@@ -77,6 +77,7 @@ void IntlTestRBNF::runIndexedTest(int32_t index, UBool exec, const char* &name,
         TESTCASE(25, TestCompactDecimalFormatStyle);
         TESTCASE(26, TestParseFailure);
         TESTCASE(27, TestMinMaxIntegerDigitsIgnored);
+        TESTCASE(28, TestNorwegianSpellout);
```

Comment:
At some point it would be nice to switch this to TESTCASE_AUTO...

Reply:
Yes, but not worth doing for small test addition like this. Better to do this for test across the board.